### PR TITLE
[OSDOCS-7122]: GCP Confidential Compute is GA (RNs)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.14

Issue:
[OSDOCS-7122](https://issues.redhat.com//browse/OSDOCS-7122)

Link to docs preview:
[Installing a cluster on GCP using Confidential VMs is generally available](https://63959--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-installation-gcp-confidential-vms)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@bscott-rh just FYI

**Comment by jeana-redhat** [comment]
Hi @eranco74, this is a GA update in the 4.14 rel notes for confidential compute on GCP. Can you PTAL for dev ack?

**Comment by jeana-redhat** [comment]
@eranco74 bumping this review request in case you missed it :)

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Comment by jeana-redhat** [comment]
@jianli-wei PTAL at this RN announcement for the change in https://github.com/openshift/openshift-docs/pull/63957

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -82,6 +82,10 @@ Deploying a three-node cluster is supported on Nutanix as of {product-title} {pr
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-three-node.adoc#installing-nutanix-three-node[Installing a three-node cluster on Nutanix].
 
+[id="ocp-4-14-installation-gcp-confidential-vms"]
+==== Installing a cluster on GCP using Confidential VMs is generally available
+In {product-title} {product-version}, using Confidential VMs when installing your cluster is generally available. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-gcp-enabling-confidential-vms_installing-gcp-customizations[Enabling Confidential VMs].
```
Comment (jianli-wei) [inline release_notes/ocp-4-14-release-notes.adoc:87]: Any PR to remove the IMPORTANT note in the linked page, which tells the "Confidential Computing is a Technology Preview feature only"? Thanks!

Comment (bscott-rh) [reply release_notes/ocp-4-14-release-notes.adoc:87]: Handled in this PR: https://github.com/openshift/openshift-docs/pull/64086

Comment (jeana-redhat) [reply release_notes/ocp-4-14-release-notes.adoc:87]: Thanks, Ben! Sorry @jianli-wei that this effort was split across two writers, I know it makes tracking more annoying :sweat_smile:

**Comment by bburt-rh** [comment]
lgtm

**Pull request merged**